web/src/app/admin/AdminConfig.tsx

```diff
@@ -56,6 +63,19 @@ const useStyles = makeStyles((theme) => ({
   saveDisabled: {
     color: 'rgba(255, 255, 255, 0.5)',
   },
+  heading: {
+    fontSize: theme.typography.pxToRem(15),
```

Comment:
Can we increase this? It feels a little odd that the text sizes for the fields are larger than the headers:
![image](https://user-images.githubusercontent.com/11381794/89455058-4f28ed00-d727-11ea-8d71-c1376b88514b.png)
It also may help to add a divider under the heading to distract away from the gap in whitespace now that the heading is inline with the card